app/scripts/lib/manifestFlags.ts

```diff
@@ -60,6 +60,18 @@ export type ManifestFlags = {
      */
     forceEnable?: boolean;
   };
+  /**
+   * Feature flags to control business logic behavior
+   */
+  remoteFeatureFlags?: {
```

Comment:
1. Include the type of `remoteFeatureFlags` in manifest as optional value. We include `testFlagForThreshold` that's created for testing purpose in [remote feature flag API](https://client-config.api.cx.metamask.io/v1/flags?client=extension&distribution=main&environment=prod) .